src/server.jl

```diff
@@ -482,7 +473,7 @@ function serve(fw::FileWatcher=SimpleWatcher(file_changed_callback);
         stop(fw)
         # close any remaining websockets
         for wss ∈ values(WS_VIEWERS), wsi ∈ wss
-            close(wsi.io)
+            close(wsi)
```

Comment:
More protocol-friendly which says we should avoid "hard closing" websocket connections, but instead initiate the CLOSE sequence before closing the underlying socket.